src/test/java/test/BitbucketOpenSearchTest.java

```diff
@@ -0,0 +1,96 @@
+package test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import test.helm.Helm;
+import test.jackson.JsonNodeAssert;
+import test.model.Product;
+
+import java.util.Base64;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static test.jackson.JsonNodeAssert.assertThat;
+import static test.model.Kind.ConfigMap;
+import static test.model.Kind.Secret;
+
+class BitbucketOpenSearchTest {
+    private Helm helm;
+
+    @BeforeEach
+    void initHelm(TestInfo testInfo) {
+        helm = new Helm(testInfo);
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = {"bitbucket"}, mode = EnumSource.Mode.INCLUDE)
+    void opensearch_statefulset_exists(Product product) throws Exception {
```

Comment:
We should also have tests for `enabled: false` && `baseUrl: value` which reflects the situation when customers are bringing their own opensearch.